penguin/system/conversation_manager.py

```
@@ -168,6 +172,30 @@ def __init__(
+                skill_runtime_config,
+                project_root=skill_project_root,
+            )
+            catalog_context = self.skill_manager.render_catalog_context()
+            if catalog_context:
+                message = self.conversation.add_context(
+                    catalog_context,
+                    source="skills_catalog",
+                )
+                message.metadata["type"] = "skills_catalog"
+                logger.info("Skills catalog autoloaded into CONTEXT")
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)